doc/develop/test/pytest.rst

```diff
@@ -91,13 +125,14 @@ keyword placed under ``harness_config`` section.
       dut.readlines_until('Hello world')
 
 shell
------
+=====
 
-Provide an object with methods used to interact with shell application.
-It calls ``wait_for_promt`` method, to not start scenario until DUT is ready.
-Note that it uses ``dut`` fixture, so ``dut`` can be skipped when ``shell`` is used.
-Scope of this fixture is determined by the ``pytest_dut_scope``
-keyword placed under ``harness_config`` section.
+Provide an `Shell <shell_class_>`_ class object with methods used to interact
+with shell application. It calls ``wait_for_promt`` method, to not start
+scenario until DUT is ready. Note that it uses ``dut`` fixture, so ``dut`` can
+be skipped when ``shell`` is used. Scope of this fixture is determined by the
```

Comment:
Done. I changed "The shell fixture inherits from..." to "The shell fixture calls...", because I think that this better describes what is done from technical point of view.